src/interfaces/filing-interfaces/filing-interface.ts

```diff
@@ -8,6 +8,7 @@ export interface IncorporationFilingIF {
       certifiedBy: string
       email: string
       date: string
+      effectiveDate: string
```

Comment:
Would it make sense to name this consistent with the same property in incorporationDateTime?

Reply:
Hmm, yes could do. Tho this is to make it consistent with the property the backend is expecting. I'll update the incorporationDateTime prop to be in line with this i think.

Reply:
K, updated. All instances of `futureEffectiveDate` are now `effectiveDate` to be congruent with the backend properties 👍